_pages/en/software-assessment-sharing-pilot.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Software Assessment Sharing Pilot
+ref: software-assessment-sharing-pilot
+lang: en
+status: In Progress;primary
+sections: Pilots
+permalink: /software-assessment-sharing-pilot.html
+---
+
+- [Problem Statement](#problem-statement)
+- [Expected Results](#expected-results)
+- [GCxchange Group](#gcxchange-group)
+- [Sharing ESDC Software Assessments](#sharing-esdc-software-assessments)
+- [Feedback](#feedback)
+- [Next Steps](#next-steps)
+
+The idea of sharing software assessments documents and results between departments has been discussed many times on GCconnex over the years but was never formally put in place.
+Some files were shared but they are not easy to find or were only shared through email.
+GCxchange is a new platform available to all GC departments and agencies, that have onboarded to M365.
+In time, it should replace GCconnex/GCmessage as the interdepartmental collaboration space for GC employees.
+The IT Strategy team at ESDC put in place a document repository on GCxchange and confirmed that the GC is one legal entity, hence sharing software assessments between departments is not a problem.
+This is a one-year pilot at ESDC and we plan to keep sharing our software assessments until August 2023.
+
+## Problem Statement
+
+GC departments are individually assessing and certifying their software assets.
+This way of working leads to many issues, including:
+
+- **Duplication/Costs** - As the same software product is assessed multiple times
+- **Scarcity of Labour** - IT specialists assessing these domains (architecture, accessibility, security, privacy, ..) exist in finite number across the GC
+- **Lower Quality** - An assessment from only one specialist (or small team) might not be as good as a wider review
+- **Employee Satisfaction** - It can be less motivating for assessors and their teams to work in isolation and make an impact only to a single department. Consequently, this leads to less learning opportunities
+- **Users’ Security and Productivity** - Bugs/issues/workarounds and security vulnerabilities are only known by the department(s) who discovered it
+
+## Expected Results
```

Comment:
After the problem statement, and before the Expected results, you should add a section "Opportunity"
This is where you essentially state that we seek to provide a place where assessors can collaborate, share their individual assessments, communicate and network.